Rubberduck.VBEEditor/Rubberduck.VBEditor.csproj

```diff
@@ -93,14 +95,14 @@
     <Compile Include="ComManagement\TypeLibs\TypeLibsAPI.cs" />
     <Compile Include="ComManagement\ReferenceEqualityComparer.cs" />
     <Compile Include="ComManagement\StrongComSafe.cs" />
-    <Compile Include="Events\ComponentEventArgs.cs" />
-    <Compile Include="Events\ComponentRenamedEventArgs.cs" />
-    <Compile Include="Events\ProjectEventArgs.cs" />
-    <Compile Include="Events\ProjectRenamedEventArgs.cs" />
-    <Compile Include="Events\SelectionChangedEventArgs.cs" />
+    <Compile Include="Events\EventArgs\ComponentEventArgs.cs" />
+    <Compile Include="Events\EventArgs\ComponentRenamedEventArgs.cs" />
+    <Compile Include="Events\EventArgs\ProjectEventArgs.cs" />
+    <Compile Include="Events\EventArgs\ProjectRenamedEventArgs.cs" />
+    <Compile Include="Events\EventArgs\SelectionChangedEventArgs.cs" />
```

Comment:
I'm not really a fan of grouping classes by what thing they are. One of the express goals of the Ducky is to allow grouping and organizing user code by something else than the type of component it happens to be in ... that said, I see this is semi-historical. Might be something to clean up in the future, though

Reply:
I wasn't really sure where else to put them. I was finding them attached to class implementations all over the place and thought they'd be easier to track down this way. I'm open to suggestions as to where they belong, but I was finding that it added a lot of cognitive load when they were in the same file as the declaring events.